### PR TITLE
fix ANR SDK-2328

### PR DESCRIPTION
**Pull request by piyush-kukadiya**
ANR was causing due to static initialization of SimpleDateFormate on main thread. It has been removed since unused in our code.

### CODE REVIEW

File: clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java [hunk 1]
```diff
@@ -49,7 +47,6 @@ public interface Constants {
     int SESSION_LENGTH_MINS = 20;
     String DEVICE_ID_TAG = "deviceId";
     String FALLBACK_ID_TAG = "fallbackId";
-    SimpleDateFormat FB_DOB_DATE_FORMAT = new SimpleDateFormat("MM/dd/yyyy", Locale.US);
```
Comment (darshanclevertap) [inline clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java:52]: @piyush-kukadiya this variable was unused?

Comment (piyush-kukadiya) [reply clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java:52]: @darshanclevertap YES. It's Facebook profile DOB related code which we removed in the past when removing google and facebook login support

Comment (darshanclevertap) [reply clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java:52]: Ok thanks!

**Review by darshanclevertap — APPROVED** [review]
👍🏾

**Pull request merged**